java/src/shared/model/ai/AIType.java

```diff
@@ -4,5 +4,6 @@
  * Created by Kyle 'TMD' Cornelison on 3/19/2016.
  */
 public enum AIType {
-    LARGEST_ARMY
+    LARGEST_ARMY,
+    RODHAMMER
```

Comment:
We can remove it, I was testing what happens with multiple AIs just to make sure